02.calendar/cal.rb

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env ruby
+require 'optparse'
+require 'date'
+
+opt = OptionParser.new
+params = {}
+
+opt.on('-m [VAL]') {|v| v }
+opt.on('-y [VAL]') {|v| v }
+opt.parse!(ARGV, into: params)
+
+if params[:m] != nil
+  month = params[:m].to_i
+else
+  month = Date.today.strftime('%m').to_i
+end
```

Comment:
`if ~ else ` のときに `!=` はあまり使いません。なぜかというと `if` と `else` を入れ替えれば `==` で書けるからです。

Reply:
こちらで`!=`ではなく`==`を使用し、順番を入れ替えました！
[Use method on Date instead of strftime](https://github.com/kmiki8242/ruby-practices/pull/2/commits/331c6a6d955b1461a44eb406846e9f8a039f16a6)